conmon-rs/server/src/rpc.rs

```diff
@@ -79,10 +79,5 @@ fn pidfile_from_params(params: &conmon::CreateContainerParams) -> capnp::Result<
 }
 
 fn path_vec_from_text_list(tl: capnp::text_list::Reader) -> Result<Vec<PathBuf>, capnp::Error> {
-    let mut v: Vec<PathBuf> = vec![];
-    for t in tl {
-        let t_str = t?.to_string();
-        v.push(PathBuf::from(t_str));
-    }
-    Ok(v)
+    tl.iter().map(|r| r.map(PathBuf::from)).collect()
```

Comment:
Random aside, but it's not just more idiomatic, it should be more efficient because
the iterator implements [ExactSizeIterator](https://docs.capnproto-rust.org/capnp/traits/struct.ListIter.html#impl-ExactSizeIterator) and the `map()` -> `collect()` chain knows to use that to allocate exactly the right sized `Vec` from the start.  One needs to do this manually with explicit iteration, i.e. use `Vec::with_capacity(tl.len()); at the start.

Reply:
I was hoping that! We may go through our code base once we have an MVP to catch other places for such optimizations.